nemo/collections/nlp/modules/common/megatron/utils.py

```diff
@@ -164,11 +164,3 @@ def autocast_forward(self, *args):
                 return self.func.apply(*args)
             else:
                 return self.func(*args)
-
-
-@contextmanager
-def dummy_handler():
-    try:
-        yield
-    finally:
-        pass
```

Comment:
Don't we use this?

Reply:
No longer, I had missed removing it in some other places and just fixed it.